runtime/v2/runc/manager/manager_linux.go

```diff
@@ -273,3 +281,63 @@ func (manager) Stop(ctx context.Context, id string) (shim.StopStatus, error) {
 		Pid:        pid,
 	}, nil
 }
+
+func (m manager) Info(ctx context.Context, optionsR io.Reader) (*task.RuntimeInfo, error) {
+	info := &task.RuntimeInfo{
+		Name: m.name,
+		Version: &task.RuntimeVersion{
+			Version:  version.Version,
+			Revision: version.Version,
+		},
+		Annotations: nil,
+	}
+	opts, err := shim.ReadRuntimeOptions[*options.Options](optionsR)
+	if err != nil {
+		if !errors.Is(err, errdefs.ErrNotFound) {
+			return nil, fmt.Errorf("failed to read runtime options (*options.Options): %w", err)
+		}
+	}
+	if opts != nil {
+		info.Options, err = protobuf.MarshalAnyToProto(opts)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal %T: %w", opts, err)
+		}
+	}
+	features, err := m.features(ctx, opts)
+	if err != nil {
+		// crun and youki do not implement `runc features` yet, at the time of writing this (May 2023)
```

Comment:
crun implements this now, since [1.8.6](https://github.com/containers/crun/releases/tag/1.8.6)

Reply:
Sorry, 1.8.7 should be used, because I fixed a bug in the formatting that caused to fail to parse the features options to the golang struct

Reply:
Fixed